assets/content/documentation/H2d/5. Displaying-Text.md

```diff
@@ -0,0 +1,65 @@
+# Displaying text
+
+`h2d.Text` can be used to draw text using bitmap fonts. It enables you to set textColor, maxWidth, textAlign, letterSpacing and lineSpacing.
+It's also possible to get the size of the text with textWidth/textHeight.
+
+### Use text in Heaps
+
+```haxe
+var tf = new h2d.Text(hxd.Res.fonts.myFontName.toFont());
+tf.text = "Hello World\nHeaps is great!";
+tf.align = Center;
```

Comment:
align is now textAlign. Also, center alignment is better defined with a set maxWidth